src/project/manifest/mod.rs

```diff
@@ -552,21 +595,48 @@ pub fn ensure_toml_target_table<'a>(
 fn get_toml_target_table<'a>(
     doc: &'a mut Document,
     platform: Option<Platform>,
+    feature: Option<&FeatureName>,
     table_name: &str,
 ) -> miette::Result<&'a mut Table> {
-    let target_table = if let Some(platform) = platform {
-        doc["target"][platform.as_str()]
-            .as_table_mut()
-            .ok_or(miette::miette!(
-                "could not find {} in {}",
-                console::style(platform.as_str()).bold(),
-                consts::PROJECT_MANIFEST,
-            ))?
-    } else {
-        doc.as_table_mut()
+    let base_table = match (feature, platform) {
```

Comment:
I think there is some duplication here that can be avoided.

First if a feature is specified get the feature table. Then if a platform is specified, find the target table.

Reply:
Fixed in: https://github.com/prefix-dev/pixi/commit/5556e0b53355a9bfd6e7fd227ac22cea19063cfc